modules/extensions-support/src/test/java/org/mule/module/extension/internal/util/IntrospectionUtilsTestCase.java

```diff
@@ -88,6 +100,36 @@ public void getNullFieldDataType() throws Exception
         IntrospectionUtils.getFieldDataType(null);
     }
 
+    @Test
+    public void getNoAnnotatedExposedPojoFields()
+    {
+        Collection<Field> exposedFields = IntrospectionUtils.getExposedFields(LifetimeInfo.class);
+        assertThat(exposedFields, is(not(empty())));
+        assertThat(exposedFields.size(), is(2));
```

Comment:
not a reason to reject the PR, but just FYI, there's a hasSize matcher in the Matchers class